vigetlabs/wordpress-site-starter
# [#60] Viget MU Plugin

# Summary

This PR ports over the main features from Vigetbase into a new MU Plugin called Viget WP.

## Issues

* #60 

## Testing Instructions

You should see the following changes to the WordPress Admin:

1. Comments Disabled
2. Viget-themed Admin Colors
3. File Editing Disabled
4. Custom Login Screen
5. Cleaned up Admin Menu & Admin Bar
6. Disabled some of the Dashboard Widgets
7. Some behind the scenes customizations to TinyMCE and ACF.

## Screenshots

<img width="1280" alt="Screenshot 2024-05-20 at 2 42 35 PM" src="https://github.com/vigetlabs/wordpress-site-starter/assets/122309362/0851d69b-4225-41d6-954b-5ab751bd9835">
